flows/events/email_sent.go

```diff
@@ -0,0 +1,41 @@
+package events
+
+import (
+	"github.com/nyaruka/goflow/flows"
+)
+
+func init() {
+	registerType(TypeEmailSent, func() flows.Event { return &EmailSentEvent{} })
+}
+
+// TypeEmailSent is our type for the email event
+const TypeEmailSent string = "email_sent"
+
+// EmailSentEvent events are created when an action has sent an email.
+//
+//   {
+//     "type": "email_sent",
+//     "created_on": "2006-01-02T15:04:05Z",
+//     "addresses": ["foo@bar.com"],
+//     "subject": "Your activation token",
+//     "body": "Your activation token is AAFFKKEE"
+//   }
+//
+// @event email_sent
+type EmailSentEvent struct {
+	baseEvent
+
+	Addresses []string `json:"addresses" validate:"required,min=1"`
```

Comment:
Since we are creating a new event, maybe we should look to the future and use `To` here? (which would allow us to add CC / BCC in the future?)

Reply:
Kinda annoying we didn't think of that when we made the action but I can change this and if we we ever add `cc`/`bcc` to the action.. we can rename `addresses` there too.. I'm hoping we can get to a point where minor flow migrations aren't a big deal.